app/src/main/java/tech/relaycorp/letro/onboarding/registration/AccountCreationProcessor.kt

```diff
@@ -0,0 +1,53 @@
+package tech.relaycorp.letro.onboarding.registration
+
+import tech.relaycorp.awaladroid.messaging.IncomingMessage
+import tech.relaycorp.letro.account.storage.AccountRepository
+import tech.relaycorp.letro.awala.AwalaManager
+import tech.relaycorp.letro.awala.processor.AwalaMessageProcessor
+import tech.relaycorp.letro.server.messages.AccountCreation
+import tech.relaycorp.letro.server.messages.InvalidAccountCreationException
+import tech.relaycorp.letro.utils.crypto.deserialiseKeyPair
+import java.util.logging.Level
+import java.util.logging.Logger.getLogger
+import javax.inject.Inject
+
+interface AccountCreationProcessor : AwalaMessageProcessor
+
+class AccountCreationProcessorImpl @Inject constructor(
+    private val accountRepository: AccountRepository,
+) : AccountCreationProcessor {
+    private val logger = getLogger(javaClass.name)
```

Comment:
In Android there is usually Singleton `Log.log()` is used for logging purposes. It's guaranteed not `null`. 

For example,` Log.w("Malformed account creation message", exc)`

Anyway, I am going to extract Logger to the abstraction to write tests later (if we need tests). So you can leave it as it is, and I'll refactor it later

Reply:
Good catch! I was going to flag the logging  I did here, but I forgot.

We're using [kotlin-logging](https://github.com/oshai/kotlin-logging) in new projects. What do you think if we use it here? Not in this PR, but when you do the refactoring.